app/config/db.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  HOST: "127.0.0.1",
+  USER: "me",
+  PASSWORD: "password",
+  DB: "api",
+  PORT: 5432,
+  dialect: "postgres",
+  pool: {
+    max: 5,
+    min: 0,
+    acquire: 30000,
+    idle: 10000
+  }
+};
```

Comment:
the credentials should be in the .env file and should not be pushed to github, but for now let's keep them like this